harold/plugins/github.py

```diff
@@ -294,6 +294,10 @@ class Salon(object):
 
         # this is just for fun
         (":tropical_fish:", ":fish:"),
+        (":trumpet::skull:", ":fish:"),
+        (":trumpet: :skull:", ":fish:"),
+        ("u\U0001F3BA \U0001F480", ":fish:"),
+        ("u\U0001F3BA\U0001F480", ":fish:"),
         (u"\U0001F420", ":fish:"),
```

Comment:
i'd like to keep the ascii+unicode versions together please. can the others move down below this one?